pylint/test/extensions/test_check_mccabe.py

```diff
@@ -0,0 +1,61 @@
+"""Tests for the pylint checker in :mod:`pylint.extensions.check_mccabe
+"""
+
+import os.path as osp
+import unittest
+
+from pylint import checkers
+from pylint.extensions.check_mccabe import McCabeMethodChecker
+from pylint.lint import PyLinter
+from pylint.reporters import BaseReporter
+
+
+class TestReporter(BaseReporter):
+
+    def handle_message(self, msg):
+        self.messages.append(msg)
+
+    def on_set_current_module(self, module, filepath):
+        self.messages = []
+
+
+class TestMcCabeMethodChecker(unittest.TestCase):
+    """Test McCabe Method Checker"""
+
+    expected_msgs = [
+        "'f1' is too complex. The McCabe rating is 1",
+        "'f2' is too complex. The McCabe rating is 1",
+        "'f3' is too complex. The McCabe rating is 3",
+        "'f4' is too complex. The McCabe rating is 2",
+        "'f5' is too complex. The McCabe rating is 2",
+        "'f6' is too complex. The McCabe rating is 2",
+        "'f7' is too complex. The McCabe rating is 3",
+        "'f8' is too complex. The McCabe rating is 4",
+        "'f9' is too complex. The McCabe rating is 9",
+        "'f10' is too complex. The McCabe rating is 11",
+        "'method1' is too complex. The McCabe rating is 1",
+        "'method2' is too complex. The McCabe rating is 18",
+        "'For 187' is too complex. The McCabe rating is 4",
```

Comment:
> for objects without a name, just say what they are, as in "This for is too complex".

Are you talking about replace the message by: 
`s/'For 187' is too complex. The McCabe rating is 4/This for is too complex. The McCabe rating is 4/g`?